src/components/scene/canvas.js

```diff
@@ -29,6 +29,8 @@ module.exports.Component = register('canvas', {
       canvas.style.height = data.height + '%';
       canvas.style.width = data.width + '%';
       scene.appendChild(canvas);
+    } else {
+      canvas.provided = true;
```

Comment:
Yeah, you could set like `data-aframe-provided`. https://github.com/aframevr/aframe/issues/798